data/tlg0627/tlg001/tlg0627.tlg001.1st1K-grc1.xml

```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
-    <teiHeader>
-    
+    <teiHeader xml:lang="eng">    
         <fileDesc>
-            <titleStmt><title>De prisca medicina</title>
+            <titleStmt><title xml:lang="lat">De prisca medicina</title>
                 <author>Hippocrates</author>
-                <editor>Émile Littré</editor>
+                <editor xml:lang="fre">Émile Littré</editor>
```

Comment:
the language is not typically applied to names. if it were Latinized, perhaps, but a modern name is the same regardless of language

Reply:
Ok, but unless the person has been published under an English or Latin or German version of his name, it seems superfluous to me. It doesn't seem like something we'd want to apply across the board except for cases of variant names. (Like a Latinized OCT name). 
I guess I'm thinking, if I'm the editor, is my name Italian?

Reply:
In this case it probably is superfluous, so I'm happy to take it out for the editor or for the editor and edition titles as you please.

Reply:
I would not remove in the case of titles. I think it's important there.